app/models/application.rb

```diff
@@ -1,2 +1,5 @@
 class Application < ApplicationRecord
+  validates :name, :email, presence: {message: 'Bitte ausfüllen!!!'}
+  validates :language_en, :language_de, presence: {message: '?'}
```

Comment:
This one is tricky. Validating the presence of boolean fields doesn't make much sense, because by definition they are always filled in (leaving them blank is also a value: `false`). What you probably wanted to validate here is that at least one of them is checked (because you won't be able to follow the workshop if you don't speak german or english). See [how we did that](https://github.com/rubymonsters/rgbapp/blob/master/app/models/application.rb#L7-L13), it's explained [here](http://guides.rubyonrails.org/active_record_validations.html#custom-methods), but we can discuss it in the next meetup.